### PR TITLE
Add instructions on how to change Leadfeeder's GTM template

### PR DESCRIPTION
**Pull request by cmihail**
Related to https://github.com/Leadfeeder/issue-tracker/issues/18381

### CODE REVIEW

**Comment by cmihail** [comment]
:white_check_mark: Review @okulik @Bajena

File: README.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
-# Leadfeeder Google Tag Manager Template
+# Leadfeeder Google Tag Manager (GTM) Template
 
 Template for [Leadfeeder](https://www.leadfeeder.com) tag in
 [Google Tag Manager](https://tagmanager.google.com)
+
+## How to change the Leadfeeder GTM template
+
+1. Motify the template code and add unit tests
```
Comment (okulik) [inline README.md]:
```suggestion
1. Modify the template code and add unit tests.
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+1. Import `template.tpl` in your own GTM account
```
Comment (okulik) [inline README.md]:
```suggestion
1. Import `template.tpl` into the existing GTM account.
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
+1. Ensure that all unit tests run successfully
```
Comment (okulik) [inline README.md]:
```suggestion
1. Make sure all unit tests run successfully.
```

File: README.md [hunk 4]
```diff
@@ -1,4 +1,17 @@
...
+1. Commit the changes
+1. Send a PR to review
```
Comment (okulik) [inline README.md]:
```suggestion
1. Create a PR and request a review.
```

File: README.md [hunk 5]
```diff
@@ -1,4 +1,17 @@
...
+1. After the PR is approved, add a change with the last commit SHA to
+   `metadata.yaml`
```
Comment (okulik) [inline README.md]:
```suggestion
1. After a PR has been approved, add the last commit's SHA with a description of the change to the `metadata.yaml` file.
```

File: README.md [hunk 6]
```diff
@@ -1,4 +1,17 @@
...
+1. Merge or ask an owner to merge your change (do NOT use squash).
+   * The change will be automatically imported by GTM using the last commit SHA.
+     For more details see https://developers.google.com/tag-manager/templates/gallery
```
Comment (okulik) [inline README.md]:
```suggestion
1. Merge or ask the owner to merge your changes (do NOT use squash). The updated template will be automatically imported by the GTM using that last commit's SHA. For more details see https://developers.google.com/tag-manager/templates/gallery.
```

File: README.md (hunk 2 above)
Comment (cmihail) [reply README.md]: I don't know if this part is clear enough. By "existing account" do we mean Leadfeeder account or developer account?
I created a new GTM account in order to write and test the changes, as I do not have access to the Leadfeeder account.

Comment (okulik) [reply README.md]: > into some existing GTM account where it could be tested

?

Comment (cmihail) [reply README.md]: Sounds good. Done

**Review by okulik — APPROVED** [review]

**Pull request merged**